docs/guides/deno/serverless-rest-api-example.mdx

```diff
@@ -0,0 +1,397 @@
+
+    // Set a JSON HTTP response
+    ctx.res.json(profile)
+  } catch (error) {
+    ctx.res.status = 404
+    ctx.res.json({
+      msg: `Profile with id ${id} not found.`,
+    })
+  }
+})
+```
+
+### Remove a profile with DELETE
+
+```typescript title:services/profiles.ts
+profileApi.delete('/profiles/:id', async (ctx) => {
+  const { id } = ctx.req.params
+
+  try {
+    // Delete a profile by id
+    await profiles.delete(id)
+
+    // Set a JSON HTTP response
+    ctx.res.json({
+      msg: `Profile with id ${id} deleted.`,
+    })
+  } catch (error) {
+    ctx.res.status = 404
+    ctx.res.json({
+      msg: `Profile with id ${id} not found.`,
+    })
+  }
+})
+```
+
+### List all profiles with GET
+
+```typescript title:services/profiles.ts
+profileApi.get('/profiles', async (ctx) => {
+  try {
+    const profilesList = []
+    // Get a profile by id
+    const keys = profiles.keys()
+
+    for await (const key of keys) {
+      const profile = await profiles.get(key)
+      profilesList.push(profile)
+    }
+
+    // Set a JSON HTTP response
+    ctx.res.json(profilesList)
+  } catch (error) {
+    ctx.res.status = 404
+    ctx.res.json({
+      msg: `Profiles not found.`,
+    })
+  }
+})
+```
+
+## Ok, let's run this thing!
+
+Now that you have an API defined with handlers for each of its methods, it's time to test it locally.
+
+```bash
+nitric start
+```
+
+Once it starts, your services will receive requests via the API port. You can use the Local Dashboard or any HTTP client to test the API. We'll keep it running for our tests.
+
+## Test the API
+
+Below are some example requests you can use to test the API. You'll need to update all values in brackets `[]` and change the URL to your deployed URL if you're testing on the cloud.
+
+### Create Profile
+
+```bash
+curl --location --request POST 'http://localhost:4001/profiles' \
+--header 'Content-Type: text/plain' \
+--data-raw '{
+    "name": "Peter Parker",
+    "age": "21",
+    "homeTown" : "Queens"
+}'
+```
+
+### Fetch Profile
+
+```bash
+curl --location --request GET 'http://localhost:4001/profiles/[id]'
+```
+
+### Fetch All Profiles
+
+```bash
+curl --location --request GET 'http://localhost:4001/profiles'
+```
+
+### Delete Profile
+
+```bash
+curl --location --request DELETE 'http://localhost:4001/profiles/[id]'
+```
+
+## Deploy to the cloud
+
+At this point, you can deploy the application to any supported cloud provider. Start by setting up your credentials and any configuration for the cloud you prefer:
+
+- [AWS](/providers/pulumi/aws)
+- [Azure](/providers/pulumi/azure)
+- [Google Cloud](/providers/pulumi/gcp)
+
+Next, we'll need to create a `stack`. Stacks represent deployed instances of an application, including the target provider and other details such as the deployment region. You'll usually define separate stacks for each environment such as development, testing and production.
+
+```bash
+nitric stack new dev
+```
+
```

Comment:
````suggestion

Continue by checking your stack file `nitric.dev.yaml` and adding in your preferred region, let's use `us-east-1`.

```yaml
# The nitric provider to use
provider: nitric/aws@latest
# The target aws region to deploy to
# See available regions:
# https://docs.aws.amazon.com/general/latest/gr/lambda-service.html
region: us-east-1
```

````